Controller/HelperController.php

```diff
@@ -110,7 +110,7 @@ public function appendFormFieldElementAction(Request $request)
         // render the widget
         // todo : fix this, the twig environment variable is not set inside the extension ...
 
-        $extension = $this->twig->getExtension('form');
+        $extension = $this->twig->getExtension('Symfony\Bridge\Twig\Extension\FormExtension');
```

Comment:
@greg0ire could you help with it?
Error `Twig_Error_Runtime: The "Symfony\Bridge\Twig\Extension\FormExtension" extension is not enabled.`
If I use `form` instead `Symfony\Bridge\Twig\Extension\FormExtension`, test passes with success.